cpp/iedriver/BrowserFactory.cpp

```diff
@@ -335,6 +335,40 @@ void BrowserFactory::LaunchBrowserUsingCreateProcess(PROCESS_INFORMATION* proc_i
   delete[] command_line;
 }
 
+bool DirectoryExists(std::wstring& dirName) {
+  DWORD attribs = ::GetFileAttributesW(dirName.c_str());
+  if (attribs == INVALID_FILE_ATTRIBUTES) {
+    return false;
+  }
+  return (attribs & FILE_ATTRIBUTE_DIRECTORY);
+}
+
+bool CreateUniqueTempDir(std::wstring &temp_dir) {
+  char temp[128];
+  std::wstring wtemp;
+
+  // get temporary folder for the current user
+  GetTempPathA(128, temp);
+  wtemp = StringUtilities::ToWString(temp);
+  if (!DirectoryExists(wtemp)) return false;
+
+  // create a IEDriver temporary folder inside the user level temporary folder
+  bool temp_dir_created = false;
+  for (int i=0; i<10; i++) {
+    std::wstring output = wtemp + L"IEDriver-" + StringUtilities::CreateGuid();
+    if (DirectoryExists(output)) continue;
+
+    CreateDirectoryW(output.c_str(), NULL);
```

Comment:
Same comment applies as previously. Should be `::CreateDirectory`.

Reply:
Updated.